### DISCUSSION TITLE
Chromium's H264 Support?

### DISCUSSION THREAD

**Discussion by arttobe** [pion/webrtc#1458] [General]
I was doing the example for an VP8 rdp stream and everything worked perfect, then I changed this line
```
	payloadType := getPayloadType(mediaEngine, webrtc.RTPCodecTypeVideo, "H264")
```
and also the stream. Now the Browser doesn't display anything and sometimes it comes an error(64000).

Here is my SDP from my stream
```
SDP:
v=0
o=- 0 0 IN IP4 127.0.0.1
s=No Name
c=IN IP4 127.0.0.1
t=0 0
a=tool:libavformat 58.29.100
m=video 5004 RTP/AVP 96
a=rtpmap:96 H264/90000
a=fmtp:96 packetization-mode=1
ffmpeg -re -f lavfi -i testsrc=size=640x480:rate=30 -vcodec libx264  -f rtp rtp://127.0.0.1:5004
```

What do I need to do for using h264 as codec, sadly I can't use VP8 in this project.

I hope someone can help me with this issue

**Comment by Sean-Der** [comment]
Chromium can only handle baseline constrained i420, also do a low keyframe interval. Also worth comparing with our [GStreamer](https://github.com/pion/example-webrtc-applications/blob/master/internal/gstreamer-src/gst.go#L57) example!

https://gist.github.com/ibc/3a10b27812d99c8abd1b is also really helpful for Chromium media pipeline debugging
